db/tasks.php

```diff
@@ -28,10 +28,10 @@
     array(
         'classname' => '\logstore_xapi\task\emit_task',
         'blocking' => 0,
-        'minute' => '*/5',
+        'minute' => '*/1',
```

Comment:
@garemoko I was saying the same thing in the comment above yours (to be precise, I was saying that I had to be sure about it, but now you are confirming it, so that's perfect). Note though that the default is not 1 minute. I changed it to 1 minute because the default is 5.